#!/usr/bin/python3
def raise_exception_msg(message=""):
    raise NameError(message)

# NameError: cuando no se encuentra una variable
